src/utils/exerciseServer.ts

```diff
@@ -15,36 +15,38 @@ const exitTimeoutMs = 5000;
 
 const argv = process.argv;
 
-if (argv.length !== 7) {
+if (argv.length !== 8) {
     console.error(`Usage: ${path.basename(argv[0])} ${path.basename(argv[1])} <project_dir> <requests_path> <server_path> <diagnostic_output> <prng_seed>`);
```

Comment:
Please update the usage text as well.

Reply:
I've added `<requests_path>` for raw error too, like replay script